ors-engine/src/main/java/org/heigit/ors/routing/RoutingProfileType.java

```diff
@@ -104,6 +104,10 @@ public static boolean isCycling(int routePref) {
                 || routePref == GH_BIKE_ROAD;
     }
 
+    public static boolean isUNKNOWN(int routePref) {
```

Comment:
I would personally refrain from introducing this type of trivial predicate method, especially if it's only used once.